smoke-test.sh

```diff
@@ -0,0 +1,176 @@
+#!/bin/bash
+set -e
+
+TARBALL_PREFIX=dotnet-sdk-
+VERSION_PREFIX=2.1
+OUTPUT_DIR=bin/x64/Release/
+DOTNET_TARBALL_REL=$(ls ${OUTPUT_DIR}dotnet-sdk-${VERSION_PREFIX}*)
+DOTNET_TARBALL=$(readlink -e ${DOTNET_TARBALL_REL})
+
+projectOutput=false
+keepProjects=false
+dotnetDir=""
+workingDir="$(pwd)"
+includeWeb=false
+
+function usage {
+    echo ""
+    echo "usage:"
+    echo "  --dotnetDir         the directory from which to run dotnet"
+    echo "  --projectOutput     echo dotnet's output to console"
+    echo "  --keepProjects      keep projects after tests are complete"
+    echo "  --includeWeb        run tests for web projects"
+    echo "  --clean             remove the testing directory if it exists"
+    echo ""
+}
+
+while :; do
+    if [ $# -le 0 ]; then
+        break
+    fi
+
+    lowerI="$(echo $1 | awk '{print tolower($0)}')"
+    case $lowerI in
+        -?|-h|--help)
+            usage
+            exit 1
+            ;;
+        --dotnetdir)
+            shift
+            dotnetDir="$1"
+            ;;
+        --projectoutput)
+            projectOutput=true
+            ;;
+        --keepprojects)
+            keepProjects=true
+            ;;
+        --includeweb)
+            includeWeb=true
+            ;;
+        --clean)
+            cleanFirst=true
+            ;;
+        *)
+            usage
+            exit 1
+            ;;
+    esac
+
+    shift
+done
+
+
+function doCommand {
+    lang=$1
+    proj=$2
+    shift; shift;
+
+    echo "starting language $lang, type $proj" | tee -a smoke-test.log
+
+    dotnetCmd=${dotnetDir}/dotnet
+    mkdir "${lang}_${proj}"
+    cd "${lang}_${proj}"
+
+    while :; do
+        if [ $# -le 0 ]; then
+            break
+        fi
+
+        echo "    running $1" | tee -a ../smoke-test.log
+
+        if [ "$1" == "new" ]; then
+            if [ "$projectOutput" == "true" ]; then
+                ${dotnetCmd} new $proj -lang $lang 
```

Comment:
I think this could `tee` to `smoke-test.log` just in case you want to go back and look at the output.  Same below.

Reply:
fixed.